tests/test_gruptree.py

```diff
@@ -144,8 +210,8 @@ def test_grupnetdf():
                 ]
             ),
             """
-└── FIELD
-    └── OP
+FIELD
+└── OP
```

Comment:
So what you want to improve in this PR is that you want to treat the top level differently? Are there more other intentions?

Reply:
Yes, the previous handling of the top-level is regarded as a bug.

The main intention of the PR was to solve a lgtm-issue, which triggered a rewrite to facilitate better testing, which again exposed the buggy top-level.

Reply:
Ah, right.